latest/src/app/documentation/demos/full-page-layouts/flow-bar-layout/flow-bar-layout.demo.ts

```diff
@@ -3,19 +3,92 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-import { Component } from "@angular/core";
-import { ClrFlowBarStep } from "@porscheinformatik/clr-addons";
+import { Component, ViewChild } from "@angular/core";
+import { ClrFlowBar, ClrFlowBarStep } from "@porscheinformatik/clr-addons";
+
+interface FlowBarDemoStep extends ClrFlowBarStep {
+    subSteps: ClrFlowBarStep[];
```

Comment:
Shouldn't this whole subStep thing, be part of the component itself? also previous/next handling below?

Reply:
yes, will move it